<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvgLumFindObjectEditWidget</class>
 <widget class="QWidget" name="AvgLumFindObjectEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>102</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="avLumL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Фильтр интенсивности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="avgLumLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ICEAVGLUMPROC_NS::TopItemWidget" name="topItemWidget" native="true"/>
   </item>
   <item>
    <widget class="ICEAVGLUMPROC_NS::ItemWidget" name="avgLumItemWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="findObjectL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Фильтр объектов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="findObjectLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ICEFINDOBJECTPROC_NS::ItemWidget" name="findObjectItemWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ICEFINDOBJECTPROC_NS::ItemWidget</class>
   <extends>QWidget</extends>
   <header>icefindobjectproc/itemwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEAVGLUMPROC_NS::TopItemWidget</class>
   <extends>QWidget</extends>
   <header>iceavglumproc/topitemwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEAVGLUMPROC_NS::ItemWidget</class>
   <extends>QWidget</extends>
   <header>iceavglumproc/itemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iceavglumfindobjectproc.qrc"/>
 </resources>
 <connections/>
</ui>
